lib/@empirica/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@empirica/core",
-  "version": "1.0.0-rc.22",
+  "version": "1.0.0-rc.23",
```

Comment:
Should the version be in 1.0.0-rc.24?

Reply:
That's an unfortunate byproduct of the current way I release. The commit that changes the version happens after the release... I will release 24 manually soon, after it's all merged into main. 👍